bin/remote-test.sh

```diff
@@ -1,7 +1,7 @@
-#!/bin/sh
+#!/bin/bash
 set -eu
 
-if [ -n "${AXIS_DEVICE_IP}" ]; then
+if [ -v AXIS_DEVICE_IP ]; then
```

Comment:
If I guessed the purpose of this commit correctly then a slightly different implementation is using `-n ${AXIS_DEVICE_IP:-}`. It works with `sh` and allows `AXIS_DEVICE_IP="" cargo ...`. Not sure if that is a feature or a bug, but I have used it to temporarily force the second branch. And `bash` is ubiquitous, so I'm fine with either.

Reply:
I think we just need to drop the `-n` with your suggestion:

```bash
if [ "${AXIS_DEVICE_IP:-}" ]; then
...
```